onnx/backend/test/case/node/split.py

```diff
@@ -17,7 +17,8 @@ def export_1d() -> None:
             'Split',
             inputs=['input'],
             outputs=['output_1', 'output_2', 'output_3'],
-            axis=0
+            axis=0,
+            num_outputs=3
```

Comment:
I think we should perhaps have test-cases for both the earlier version and newer version? @jcwchen ?

Reply:
After this PR: https://github.com/onnx/onnx/pull/3855, going forward all of node test cases for the same operator will be updated if there is an opset bump. If people would like to keep some old test cases for old opset versions, they need to specify the opset_version while creating node test cases. Take Loop op as an example, see this [line](https://github.com/onnx/onnx/blob/ea7c5204713cce87e8c22e5382833c68483f329b/onnx/backend/test/case/node/loop.py#L127).

Reply:
I applied the changes you suggested. Opset13 tests were reverted, new (opset18) tests stayed the same.